tests/FSharpVSPowerTools.Core.Tests/OutliningTests.fs

```diff
@@ -38,128 +41,133 @@ let (=>) (source: string) (lineRanges: (int * int) list) =
             let multilineRanges =
                 getOutliningRanges tree
                 |> Seq.filter (fun r -> r.StartLine <> r.EndLine)
-                |> Seq.map (fun r -> r.StartLine, r.EndLine)
+                |> Seq.map (fun r -> r.StartLine, r.StartColumn, r.EndLine, r.EndColumn)
                 |> List.ofSeq
-            CollectionAssert.AreEquivalent (List.sort lineRanges, List.sort multilineRanges)
+            CollectionAssert.AreEquivalent (List.sort ranges, List.sort multilineRanges)
```

Comment:
I would just name them `expectedRanges` and `actualRanges` to avoid confusion.

`CollectionAssert` members don't give good error messages on failing tests. I think our `assertEqual` could do better jobs here.